common-arcana.lic

```diff
@@ -110,10 +110,10 @@ module DRCA
     match
   end
 
-  def ritual(spell, ignored_npcs)
-    DRC.retreat(ignored_npcs) unless spell['skip_retreat']
+  def ritual(spell, settings)
```

Comment:
❓ Won't this break for anyone that gets the updated `common-arcana` before we merge in `combat-trainer` changes?

I.e. their `combat-trainer` will be sending in `ignored_npcs` where this function will be expecting `settings` and get `nil` when it tries to pull out `ignored_npcs` or `default_stance`.

I'd rather see a third argument here.

Reply:
Yes it will and I talked to a few people in slack about it.  They suggested that this was a better method of implementing the change and more inline with the way the rest of the common files implement methods.

Reply:
There was also talk recently about finding some way of forcing common method updates to propagate to people.

Reply:
A third argument will also break combat trainer because the number of arguments won't match.  That happened with the last change I made to combat trainer and equipmanager.

https://github.com/rpherbig/dr-scripts/pull/3661
https://github.com/rpherbig/dr-scripts/pull/3662